pom.xml

```diff
@@ -41,6 +41,10 @@
             <artifactId>dropwizard-core</artifactId>
         </dependency>
         <dependency>
+			<groupId>com.twilio.sdk</groupId>
+			<artifactId>twilio</artifactId>
+	    	<version>(7.0,7.9)</version>
```

Comment:
can you switch this to use latest? https://mvnrepository.com/artifact/com.twilio.sdk/twilio